tests/integration/conftest.py

```diff
@@ -209,19 +210,11 @@ def noised_sample_data_taxes_1040(config: dict[str, Any]) -> pd.DataFrame:
 
 
 def get_unnoised_data(dataset_name: str) -> Dataset:
-    result = _initialize_dataset_with_sample(dataset_name)
+    result = initialize_dataset_with_sample(dataset_name)
     result.data = coerce_dtypes(result.data, result.dataset_schema)
     return result
 
 
-def _initialize_dataset_with_sample(dataset_name: str) -> Dataset:
```

Comment:
This got moved to tests/utilities.py since it's shared across /integration and /release.